packages/examples/cvat/exchange-oracle/alembic/versions/1707173682_non_unique_escrows_c1e74c227cfe.py

```
@@ -22,6 +35,22 @@ def upgrade() -> None:
 
 
 def downgrade() -> None:
+    offline_mode = op.get_context().environment_context.is_offline_mode()
+    assert offline_mode or "TESTING" in os.environ or "test" in op.get_bind().engine.url, (
```

This is a configuration error, so it's better to throw a regular exception.